upstream-community-operators/elastic-cloud/0.8.0/elastic-cloud-eck.clusterserviceversion.yaml

```diff
@@ -0,0 +1,252 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: elastic-cloud-eck.v0.8.0
+  namespace: placeholder
+  annotations:
```

Comment:
Please take a look at our required fields doc for additional annotations: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md